autoload/neomake.vim

```diff
@@ -71,7 +76,14 @@ function! neomake#MakeJob(maker) abort
     if append_file
         call add(args, '%:p')
     endif
-    call map(args, 'expand(v:val)')
+    
+    echom string(args)
+    if neomake#utils#IsRunningWindows()
+        " Don't expand '/C' argument to cmd.exe
+        call map(args, 'v:val !=? "/C" ? expand(v:val) : v:val')
```

Comment:
I'll need to have a deeper look but the whole issue with /C being turned into \C is that Neovim quotes the command when it is called using jobstart([...]) - more in https://github.com/neovim/neovim/pull/3812